File: test1.js
function isPalindrom(word) {
    word = word.toLowerCase();
    for (let i = 0; i < Math.floor(word.length / 2); i++) {
        if (word[i] !== word[word.length - i - 1]) {
            return false
        }
    }
    return true
}
var word = 'Deleveled';
console.log(word, isPalindrom(word));

File: test1.go
package main

import (
	"fmt"
	"strings"
	"math"
)

func isPalindrom(word string) bool {
    word = strings.ToLower(word)
    for i := 0; float64(i) < math.Floor(float64(len(word) / 2)); i++ {
        if (word[i] != word[len(word) - i - 1]) {
            return false
        }
    }
    return true
}

func main() {
	word := "Deleveled"
	fmt.Println(word, isPalindrom(word))
}
